### PR TITLE
Many many vehicle to vehicle types

### PR DESCRIPTION
**Pull request by Sleepingwell**
Pedro: Could you please check that this suits.

Note that you might want to use the test db:

- DB name: cvts_test
- user name: cvts_test

### CODE REVIEW

File: cvts/models.py [hunk 1]
```diff
@@ -13,11 +14,24 @@
...
+    Column('vehicle_id', Integer, ForeignKey('vehicles.id')),
+    Column('vehicle_type_id', Integer, ForeignKey('vehicle_types.id'))
+)
+
 class Vehicle(DBase):
     """Data on a vehicle."""
     __tablename__ = 'vehicles'
     id            = Column(Integer, primary_key=True)
-    rego          = Column(String)
+    vehicle_types = relationship("VehicleType", secondary=vehicle_type_association_table)
```
Comment (pedrocamargo) [inline cvts/models.py:26]: I am probably missing something, but vehicle type is not unique

Comment (pedrocamargo) [reply cvts/models.py:26]: See my mesage on Slack. Vehicle type may as well be unique if we disregard "unclassified"

Comment (Sleepingwell) [reply cvts/models.py:26]: The table defined by *vehicle_type_association_table* (table *vehicle_vehicle_type_assoc* in the DB) contains the many2many.

**Comment by Sleepingwell** [comment]
After discovering that in every case of duplicates, one of them is 'unidentified', we decided to go with a one-to-many.

**Pull request closed without merging**